docker-compose.yml

```diff
@@ -1,27 +1,25 @@
 version: "3.7"
 
+volumes:
+  sbtuser:
+
 services:
   book_server: #container name
     build:
       context: .
       dockerfile: Dockerfile
     volumes:
       - ./:/opt/book-api
-    ports: 
+      - sbtuser:/home/sbtuser
```

Comment:
The changes to the dockerfile cache the dependencies in the docker image. When you run the image, if you change dependencies they will be cached in this volume.

Together, that should mean much less downloading.